### ISSUE TITLE
binding power of "at" and ";"

### ISSUE THREAD

**Issue by fpvandoorn** [leanprover/lean3#546]
In the following example I cannot remove parentheses:

``` lean
  definition square_of_eq (r : p₁₀ ⬝ p₂₁ = p₀₁ ⬝ p₁₂) : square p₁₀ p₁₂ p₀₁ p₂₁ :=
  by cases p₁₂; (esimp [concat] at r); cases r; cases p₂₁; cases p₁₀; exact ids
```

Can we allow this?

**Comment by leodemoura** [comment]
This will be fixed after #461 is closed.

**Comment by leodemoura** [comment]
This should work now. 
Example: https://github.com/leodemoura/lean/blob/a23118d357d08255652ec64fe90f62b58f5f9fa0/tests/lean/run/by_exact.lean
